### DISCUSSION TITLE
Can't Scan VPN Subnet, Can Add Hosts Manually

### DISCUSSION THREAD

**Discussion by jakendrick3** [seriousm4x/UpSnap#708] [Help wanted]
Hi, I'm currently away from home and running Upsnap on a Raspberry Pi (Dietpi). I'm able to scan the subnet of the network I'm on just fine, but my wireguard tunnel back to home returns no hosts when I try to scan the subnet that it's configured to. Running nmap on the host for that subnet returns all expected results, and manually adding a host from the VPN subnet also works as expected. Any ideas for getting the network scan to run?

**Comment by seriousm4x** [comment]
Not an expert on this but you might need to raise the `UPSNAP_SCAN_TIMEOUT=500ms` timeout since you are away from your home network. Maybe try 2000ms or something like that.
